apps/daffio/src/app/docs/api/components/api-list-children/api-list-children.component.html

```diff
@@ -0,0 +1,6 @@
+@for (childDoc of children; track childDoc.id) {
```

Comment:
can we name this `api-list-section` instead?

Reply:
sure, feel free to do this during your ui updates